app/views/submissions.scala.html

```diff
@@ -27,6 +27,8 @@
     appConfig = appConfig,
     bodyClasses = None) {
 
+    @components.back_link("choice")
```

Comment:
To be consistent with other pages my propose is to use `/customs-declare-exports/choice` as a back link

Edit: or like below... with `routes.ChoiceController.displayChoiceForm().url` if it's possible. This option looks good

Reply:
I didnt realise the argument was a href, I'll fix